x-pack/solutions/search/plugins/search_indices/public/code_examples/javascript.ts

```diff
@@ -149,3 +149,41 @@ const updateMappingResponse = await client.indices.putMapping({
 });
 console.log(updateMappingResponse);`,
 };
+
+export const JSSemanticIngestDataExample: IngestDataCodeDefinition = {
+  ...JSIngestDataExample,
+  ingestCommand: ({
+    apiKey,
+    elasticsearchURL,
+    sampleDocuments,
+    indexName,
+  }) => `const { Client } = require('@elastic/elasticsearch');
```

Comment:
I updated all of the imports to use `require()` syntax to make this copy/paste directly into node v20

Reply:
full ESM support is coming, but I am but one person so it hasn't happened yet. :smile: